lib/resources/example_json/r4_examples_practitioner.rb

```diff
@@ -166,5 +166,46 @@ module Resources
       ]
     }
 
+    R4_PRACTITIONER_CREATE ||= {
+      "resourceType": "Practitioner",
+      "active": true,
+      "name": [
+        {
+          "family": "Williams",
+          "given": [
+            "Rory",
+            "James"
+          ],
+          "prefix": [
+            "Dr."
+          ],
+          "suffix": [
+            "M.D."
+          ],
+          "period": {
+          	"start": "2019-12-01T00:00:00.000Z"
+          }
+        }
+      ],
+      "identifier": [
+        {
+          "type": {
+            "coding": [
+              {
+                "code": "DEA",
+                "system": "http://terminology.hl7.org/CodeSystem/v2-0203"
+              }
+            ]
+          },
+          "system": "urn:oid:1.1.1.1.1.1",
```

Comment:
We should use the real system that we associate with DEA, urn:oid:2.16.840.1.113883.4.814 

Also, let's have the example match the DEA number format: https://en.wikipedia.org/wiki/DEA_number

Reply:
Used correct system and applied an actual DEA number. 

https://github.com/cerner/fhir.cerner.com/pull/330/commits/c128b759084993b859b499ef4e9e83aedf8a4898#diff-b7b6f9bc4e73a723f2d414890a03bde8R200

The current format of the DEA Number linked actually has an incorrect calculation for the checksum on the last number (had it listed as 1234567, when it should be 1234563), so I edited the wikipedia entry too 😅